pkg/dashboard/templates/email.gohtml

```diff
@@ -0,0 +1,26 @@
+{{ define "email" }}
+<div class="email-box" id="email-box">
+  <details open>
+    <summary class="--hideMarker">
+      <h2 class="email-box__toggle-title">
+        Add cost estimates to your Goldilocks dashboard!
+        <i aria-hidden="true" class="fas fa-fw fa-angle-down --showWhenClosed"></i
+        ><i aria-hidden="true" class="fas fa-fw fa-angle-up --showWhenOpen"></i>
+      </h2>
+    </summary>
+
+    <section class="email-box__content">
```

Comment:
I updated the code based on your feedback. Thank you

Reply:
Thank you!

Just in case you don't know about them, it may be helpful to add `action` and `method` attributes to this form. I can see you're still working on the functionality to actually process the form data, so it makes sense not to have these in yet, but they may help with building out that processing functionality later. More info, in case it helps: [MDN Docs: Sending form data](https://developer.mozilla.org/en-US/docs/Learn/Forms/Sending_and_retrieving_form_data).